# HG changeset patch
# User Sean Farley <sean.michael.farley@gmail.com>
# Date 1397849684 18000
#      Fri Apr 18 14:34:44 2014 -0500
# Node ID 0c556df11dafc4475317fdc3598c7db1ad0b4265
# Parent  6bac5a7664f423bf62146ffd002bff8c397df3ec
gildas-sh: use variables in script

diff --git a/admin/gildas-env.sh b/admin/gildas-env.sh
--- a/admin/gildas-env.sh
+++ b/admin/gildas-env.sh
@@ -416,11 +416,11 @@ EOF
         unset PYTHON_INC_DIR PYTHON_LIB_DIR
         unset NUMPY_PRESENT  NUMPY_INC_PRESENT NUMPY_INC_DIR
     fi
     #
     # Check availability of g++. This only has an impact on ATM2009
-    if \which g++ > /dev/null 2>&1; then
+    if \which @CXX@ > /dev/null 2>&1; then
         ATM2009_PRESENT=yes
     else
         gagenv_message "g++ not found in your \$PATH"
         gagenv_message "This means that ATM 2009 will not be available"
         ATM2009_PRESENT=no
@@ -433,38 +433,38 @@ EOF
         PYTHON_INC=no
         PYTHON_LIB=no
         NUMPY_PRESENT=no
         NUMERIC_PRESENT=no
         SQLITE3_PRESENT=no
-        if \which python > /dev/null 2>&1; then
+        if \which @PYTHON@ > /dev/null 2>&1; then
             # Check the version of python the user is currently using
-            if python -V 2>&1 > /dev/null 2>&1; then
-        	PY_VERS_FULL=`python -V 2>&1 | cut -d' ' -f2`
+            if @PYTHON@ -V 2>&1 > /dev/null 2>&1; then
+        	PY_VERS_FULL=`@PYTHON@ -V 2>&1 | cut -d' ' -f2`
         	PY_VERS=`echo $PY_VERS_FULL | sed "s%\([0-9]\.[0-9][0-9]*\).*%\1%"`
         	# Search for Python header and library. kernel/python/binding/setup.py
         	# does exactly the same, so this should be consistent
-        	PYTHON_INC_DIR=`python -c "import distutils.sysconfig; print(distutils.sysconfig.get_config_var('INCLUDEPY'))"`
+        	PYTHON_INC_DIR=`@PYTHON@ -c "import distutils.sysconfig; print(distutils.sysconfig.get_config_var('INCLUDEPY'))"`
         	if [ -e "$PYTHON_INC_DIR/Python.h" ]; then PYTHON_INC=yes; else PYTHON_INC=no; fi
-        	PYTHON_LIB_DIR=`python -c "import distutils.sysconfig; print(distutils.sysconfig.get_config_var('LIBDIR'))"`
+        	PYTHON_LIB_DIR=`@PYTHON@ -c "import distutils.sysconfig; print(distutils.sysconfig.get_config_var('LIBDIR'))"`
         	PYTHON_LIB=yes  # libpython name on the various supported systems is just
         	                # unpredictible. Can not check in advance if the linker will
         	                # find it or not.
         	#
         	# Now search for Numpy or Numeric
-        	if python -c "import numpy" > /dev/null 2>&1; then
+        	if @PYTHON@ -c "import numpy" > /dev/null 2>&1; then
         	    NUMPY_PRESENT=yes
-        	    NUMPY_INC_DIR=`python -c "import numpy; print(numpy.__path__[0] + '/core/include')"`
+        	    NUMPY_INC_DIR=`@PYTHON@ -c "import numpy; print(numpy.__path__[0] + '/core/include')"`
         	    if [ -e "$NUMPY_INC_DIR/numpy/arrayobject.h" ]; then
         		NUMPY_INC_PRESENT=yes
         	    else
         		NUMPY_INC_PRESENT=no
         	    fi
-        	elif  python -c "import Numeric"> /dev/null 2>&1; then
+        	elif  @PYTHON@ -c "import Numeric"> /dev/null 2>&1; then
         	    NUMERIC_PRESENT=yes
         	fi
         	# Search for sqlite3
-        	if python -c "import sqlite3" > /dev/null 2>&1; then
+        	if @PYTHON@ -c "import sqlite3" > /dev/null 2>&1; then
         	    SQLITE3_PRESENT=yes
         	fi
             fi
         else
             gagenv_message "Python not found in your \$PATH"
@@ -536,29 +536,29 @@ EOF
         fi
         if file_present "libatm2003." "${DIR}"; then
             ATM2003_PRESENT=yes
             ATM2003_LIB_DIR=$DIR
         fi
-        if file_present "liblapack." "${DIR}"; then
+        if file_present "libLAPACK." "${DIR}"; then
             LAPACK_PRESENT=yes
             LAPACK_LIB_DIR=$DIR
         fi
-        if file_present "libblas." "${DIR}"; then
+        if file_present "libBLAS." "${DIR}"; then
             BLAS_PRESENT=yes
             BLAS_LIB_DIR=$DIR
         fi
         if file_present "libslatec." "${DIR}"; then
             SLATEC_PRESENT=yes
             SLATEC_LIB_DIR=$DIR
         fi
-        if file_present "libfftw3." "${DIR}"; then
-            FFTW3_PRESENT=yes
-            FFTW3_LIB_DIR=$DIR
-            if file_present "libfftw3f." "${DIR}"; then
-        	FFTW3F_PRESENT=yes
-            fi
-        fi
+#        if file_present "libfftw3." "${DIR}"; then
+#            FFTW3_PRESENT=yes
+#            FFTW3_LIB_DIR=$DIR
+#            if file_present "libfftw3f." "${DIR}"; then
+#        	FFTW3F_PRESENT=yes
+#            fi
+#        fi
         if file_present "libasdmStandalone." "${DIR}"; then
             SDM_PRESENT=yes
             if [ "$SDM_PRESENT" = "yes" ]; then
         	SDM_LIB_DIR=$DIR
             fi
@@ -588,10 +588,12 @@ EOF
     IFS=$OLDIFS
     #
     XML_PRESENT="yes"
     #
     unset GAG_LIB_DEP_PATH GAG_INC_DEP_PATH
+    GAG_INC_DEP_PATH=$GAG_INC_DEP_PATH:@PREFIX@/include
+    GAG_LIB_DEP_PATH=$GAG_INC_DEP_PATH:@PREFIX@/lib
     #
     if [ "$X11_INC" = "yes" ]; then
         gagenv_message "Found X11 header  in $X11_INC_DIR"
         GAG_INC_DEP_PATH=$GAG_INC_DEP_PATH:$X11_INC_DIR
     else
